pkgs/tools/backup/restic/rest-server.nix

```diff
@@ -13,10 +13,7 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-8x5qYvIX/C5BaewrTNVbIIadL+7XegbRUZiEDWmJM+c=";
 
-  preCheck = ''
-    substituteInPlace cmd/rest-server/main_test.go \
-      --replace "/tmp/restic" "/build/restic"
-  '';
+  patches = [ ./0002-Backport-rest-server-tests-os-TempDir.patch ];
```

Comment:
Backported from where? Can we fetch the patch instead?

Reply:
from master branch of project's repo. they didn't cut a release newer than what we work with here, 0.11.0.

Reply:
ooh. now i get what you're asking.
yeah, we can just fetch this:
https://github.com/restic/rest-server/commit/a87a50ad114bdaddc895413396438df6ea0affbb.patch

let me update.